component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/emailotp/EmailOTPAuthenticatorConstants.java

```diff
@@ -20,6 +20,8 @@
 package org.wso2.carbon.identity.authenticator.emailotp;
 
 public class EmailOTPAuthenticatorConstants {
+    public static final String CHAR_SET_UTF_8 = "UTF-8";
```

Comment:
add new line before this

Reply:
no need to define this constant. we can use StandardCharsets.UTF_8.name()

Reply:
Fixed this issue